src/components/BackgroundImg.js

```diff
@@ -3,10 +3,12 @@ import styled, { css } from "styled-components"
 const BackgroundImg = styled.div`
   width: ${({ width }) => width};
   height: ${({ height }) => height};
-  background-repeat: no-repeat;
+  background: ${ 
+    ({ src, gradient }) => gradient 
+    ? `${gradient} , url(${src}) no-repeat center`
```

Comment:
you might consider writing this all as one line, something like:
```suggestion
    `${ gradient + ' ,'  || '' } url(${src}) no-repeat center`
```
though you might think it's less readable which is valid.
(Really I just wanted to test out this cool new suggestions feature!)

Reply:
Suggestions feature is great! I can commit all your recommendations so easily! 😍